src/layers/VolumeLayer/VolumeLayer.js

```diff
@@ -100,6 +104,9 @@ const VolumeLayer = class extends CompositeLayer {
         loaderSelection.length;
       const onUpdate = () => {
         progress += 0.5 / totalRequests;
+        if (this.props.onUpdate) {
```

Comment:
Not clear to me why the if statement. By default it is a noop function that is always called.

Reply:
Figured better to check.  If someone passes in an `undefined` react prop or something, it would fail so seemed better to check first.

Reply:
I had run into not being careful about this before - if any sort of prop is passed in, the default is not used, even if that prop is undefined.